kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/catalogManaged/CatalogManagedEnablementSuite.scala

```diff
@@ -46,42 +42,51 @@ class CatalogManagedEnablementSuite extends AnyFunSuite with TestUtils {
       expectedCatalogManagedSupported: Boolean = false)
 
   val catalogManagedTestCases = Seq(
-    CatalogManagedEnablementTestCase(
-      testName = "CREATE: catalogManaged enablement flag => enables catalogManaged and ICT",
+    // ===== CREATE cases =====
+    CatalogManagedTestCase(
+      testName = "CREATE: set catalogManaged=supported => enables catalogManaged and ICT",
       operationType = "CREATE",
       transactionProperties = Map("delta.feature.catalogOwned-preview" -> "supported"),
       expectedSuccess = true,
       expectedIctEnabled = true,
       expectedCatalogManagedSupported = true),
-    CatalogManagedEnablementTestCase(
-      testName = "UPDATE: catalogManaged enablement flag => enables catalogManaged and ICT",
+    CatalogManagedTestCase(
+      testName = "ILLEGAL CREATE: set catalogManaged=supported and explicitly disable ICT => THROW",
+      operationType = "CREATE",
+      transactionProperties = Map(
+        "delta.feature.catalogOwned-preview" -> "supported",
+        "delta.enableInCommitTimestamps" -> "false"),
+      expectedSuccess = false,
+      expectedExceptionMessage =
+        Some("Cannot disable inCommitTimestamp when enabling catalogManaged")),
+
+    // ===== UPDATE cases =====
+    CatalogManagedTestCase(
+      testName = "UPDATE: set catalogManaged=supported => enables catalogManaged and ICT",
       operationType = "UPDATE",
       initialTableProperties = Map.empty, // Start with basic table
       transactionProperties = Map("delta.feature.catalogOwned-preview" -> "supported"),
       expectedSuccess = true,
       expectedIctEnabled = true,
       expectedCatalogManagedSupported = true),
-    CatalogManagedEnablementTestCase(
-      testName = "UPDATE: catalogManaged enablement flag => enables ICT if previously disabled",
+    CatalogManagedTestCase(
+      testName = "UPDATE: set catalogManaged=supported => enables ICT if previously disabled",
       operationType = "UPDATE",
       initialTableProperties = Map("delta.enableInCommitTimestamps" -> "false"),
       transactionProperties = Map("delta.feature.catalogOwned-preview" -> "supported"),
       expectedSuccess = true,
       expectedIctEnabled = true,
       expectedCatalogManagedSupported = true),
-    CatalogManagedEnablementTestCase(
```

Comment:
Just moved this up. Also, I clean up some of the test case names to be more succinct and clearer